sdks/builds/android.mk

```diff
@@ -351,4 +351,5 @@ $(eval $(call AndroidCrossMXETemplate,cross-arm64-win,x86_64,aarch64-v8a,arm64-v
 $(eval $(call AndroidCrossMXETemplate,cross-x86-win,i686,i686,x86,llvm-llvmwin32,i686-none-linux-android))
 $(eval $(call AndroidCrossMXETemplate,cross-x86_64-win,x86_64,x86_64,x86_64,llvm-llvmwin64,x86_64-none-linux-android))
 
-$(eval $(call BclTemplate,android,monodroid monodroid_tools,monodroid monodroid_tools))
+$(eval $(call BclTemplate,android,monodroid monodroid_tools monodroid_designer,monodroid monodroid_tools))
```

Comment:
You want to remove `monodroid_designer` from this line then, and I'll remove it on my PR too